src/windshaft/dashboard.js

```diff
@@ -56,17 +56,18 @@ WindshaftDashboard.prototype._createInstance = function(options) {
         urls: dashboardInstance.getTiles('mapnik')
       });
 
-      this._updateWidgetURLs({
-        layerId: options.layerId
-      });
-
       // update other kind of layers too
-      this.layers.each(function(layer) {
+      this.layers.each(function(layer, layerIndex) {
         if (layer.get('type') === 'torque') {
+          layer.set('meta', dashboardInstance.getLayerMeta(layerIndex));
```

Comment:
Yes, that kind of information is only provided for torque layers, and at the same time is required by time-series widget associated to torque layers.

We need to review this use case when using time-series widget with non-torque layers.